### PR TITLE
Allow caller of readFile to handle errors.

### PR DESCRIPTION
**Pull request by tobli**
Instead of always alert()-ing errors, have caller decide by using a (error, data) callback pattern.
NOTE: This is a change in the public api.

### CODE REVIEW

**Comment by tobli** [comment]
Maybe we should actually return a Promise instead.

File: src/index.html [hunk 1]
```diff
@@ -75,9 +75,9 @@
           }
 
           // Just needed for gzipped *.zhar files, you can use the standard FileReader api for normal .har files
-          perfCascadeFileReader.readFile(files[0], evt.target.value, function(data){
-            if(!data){
-              console.error("Can't read file")
+          perfCascadeFileReader.readFile(files[0], evt.target.value, function(error, data){
+            if(error){
+              console.error(e)
```
Comment (micmro) [inline src/index.html]: Is this a typo? Should it be `console.error(error)`?

Comment (tobli) [reply src/index.html]: Good catch! Yeah, that's a typo. I just pushed an update.

**Comment by micmro** [comment]
Great - feel free to merge whenever you're ready!

**Pull request merged**